src/xrCore/xr_ini.cpp

```diff
@@ -451,13 +451,22 @@ void CInifile::Load(IReader* F, pcstr path, allow_include_func_t allow_include_f
             R_ASSERT(path && path[0]);
             if (_GetItem(str, 1, inc_name, '"'))
             {
+#ifdef XR_PLATFORM_WINDOWS
```

Comment:
This is ridiculous, please do something with it. `xr_fs_nostrlwr` is already a macro...

Reply:
@Xottab-DUTY fixed